hypershift-operator/controllers/hostedcluster/metrics.go

```diff
@@ -51,6 +51,12 @@ var (
 		Help: "Indicates cluster proxy state for each cluster",
 		Name: ProxyName,
 	}, []string{"namespace", "name", "proxy_http", "proxy_https", "proxy_trusted_ca"})
+
+	UnremovalAWSObjectsName            = "hypershift_cluster_skipped_object_deletion"
+	hostedClusterSkippedObjectDeletion = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Help: "Times the controller tried to delete a AWS Cloud resource without luck",
```

Comment:
for starters can we just keep it simple an have a metric that is 1 one deletion is skipped?